internal/model/view.go

```diff
@@ -227,12 +229,22 @@ func (m *Model) refreshViewportContent() {
 		rawSuffix := ""
 		if node.IsDir {
 			if count := dirSelectedCounts[node.Path]; count > 0 {
-				rawSuffix = fmt.Sprintf(" [%d]", count)
+				rawSuffix = fmt.Sprintf(" [%d 📄]", count)
```

Comment:
Let's revert this change:

```suggestion
				rawSuffix = fmt.Sprintf(" [%d]", count)
```

An icon here will be handled in a future PR, tracked in issue https://github.com/epilande/codegrab/issues/10.